codeql_javascript_dataset/CWE-094/examples/code_injection_eval.js

```
@@ -0,0 +1,7 @@
+function calculateExpression(userInput) {
+    // Extremely dangerous - never do this
+    const result = eval(userInput);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)